README.md

```diff
@@ -15,7 +15,7 @@ If your app suppots multiple platforms, the API calls to this plugin should be p
 
 ```javascript
 if (cordova.platform === 'android') {
```

Comment:
Not sure when the platform guard was added but, the readme is wrong.

iOS results in what is almost a no-op. You can use apis without platform guards with a single cavet - iOS' inset object will always contain 0 for the 4 sides. It is expected that you will use const and env css variables.

This behaviour is almost identical to current master. In current master, you can add listeners on iOS but they will never be called. The only time you had to think about platform guards was if you used the `getInsets` API. Noting this API was on the plugin object, not an Inset class.

This was choice was intentional so that we could explicitly avoid using platform guards in our apps.

This choice was for convenience so if it's problematic or in the way we could get rid of this idea, though I suppose that would be a breaking change as it could break the iOS side if people didn't use platform guards before...

Whether it's in this PR or another PR, we should decide the expectations of the plugin so that everyone is on the same page.

Reply:
imo best to keep it documented as if it's unsafe to call on the API on iOS. I know it is guarded but it still isn't really right.

Reply:
With the changes coming in this PR, it's probably better to expect people to platform guard as iOS will receive an useless Insets object and inline style could override their actual css env vars usage.